pdr_backend/lake/csv_data_store.py

```diff
@@ -123,8 +123,8 @@ def __init__(self, base_path: str, table_name: str):
         self.table_name = table_name
 
     @staticmethod
-    def from_table(table):
-        return CSVDataStore(table.base_path, table.table_name)
+    def from_table(table, ppss):
+        return CSVDataStore(ppss.lake_ss.lake_dir, table.table_name)
```

Comment:
I'm seeing ppss being passed around so much.
It's growing too.
And it's a configuration file.

Isn't there a better way?